packages/datadog-plugin-jest/test/circus.spec.js

```diff
@@ -167,7 +167,7 @@ describe('Plugin', function () {
             expect(testSpan.service).to.equal('test')
             expect(testSpan.resource).to.equal(`packages/datadog-plugin-jest/test/jest-test.js.${name}`)
             expect(testSpan.meta[TEST_FRAMEWORK_VERSION]).not.to.be.undefined
-          }, { timeoutMs: testTimeout })
+          }, { timeoutMs: testTimeout / 2 })
```

Comment:
Seems weird to use a variable if it's divided on every use.

Reply:
well `this.timeout` is called with `testTimeout`, but right